config/routes.rb

```diff
@@ -2,4 +2,9 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :places
   resources :varieties
+  resources :orders, only: [:index, :new, :create]
```

Comment:
Me parece que "orders#index" podría ser la ruta raíz para toda la app, no? 
Ahora sigue apareciendo el "you're on rails":
![image](https://user-images.githubusercontent.com/530965/54223235-cb3ce800-44d5-11e9-9e8c-81836a0e8805.png)

Reply:
Se corregirá en otro PR que ordena las rutas